From a0a9fed68126eb24e04af924c0d46351ff5eeb0f Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Mon, 26 Jan 2015 09:55:04 -0300
Subject: [PATCH] thread/Name: include stdio.h for prctl as well

We're still using snprintf so we need it, otherwise it leads to build
failure.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
 src/thread/Name.hxx | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/thread/Name.hxx b/src/thread/Name.hxx
index a99208d..8f9d7d3 100644
--- a/src/thread/Name.hxx
+++ b/src/thread/Name.hxx
@@ -25,6 +25,7 @@
 #  include <pthread.h>
 #elif defined(HAVE_PRCTL)
 #  include <sys/prctl.h>
+#  include <stdio.h>
 #  ifdef PR_SET_NAME
 #    define HAVE_THREAD_NAME
 #  endif
-- 
2.6.4

